src/Swagger/Serializer/DocumentationNormalizer.php

```diff
@@ -455,20 +458,28 @@ private function getPropertySchema(PropertyMetadata $propertyMetadata) : \ArrayO
         return $propertySchema;
     }
 
-    private function computeDoc(Documentation $object, \ArrayObject $definitions, \ArrayObject $paths): array
+    /**
+     * Computes the Swagger documentation.
+     *
+     * @param Documentation $documentation
+     * @param \ArrayObject  $definitions
+     * @param \ArrayObject  $paths
+     *
+     * @return array
+     */
+    private function computeDoc(Documentation $documentation, \ArrayObject $definitions, \ArrayObject $paths) : array
     {
         $doc = [
             'swagger' => self::SWAGGER_VERSION,
-            'info' => ['title' => $object->getTitle()],
+            'basePath' => $this->urlGenerator->generate('api_entrypoint'),
+            'info' => [
+                'title' => $documentation->getTitle(),
+                'version' => $documentation->getVersion(),
+                'description' => $documentation->getDescription(),
```

Comment:
The default value of the description is the empty string, and we don't want to use that, do we?